install-and-launch.sh

```diff
@@ -429,6 +431,37 @@ launch_sublime() {
 }
 
 install_sublime() {
+    if [ -z "$clone_platform" ]; then
+        clone_platform=true
+    fi
+
+    if [ "$clone_platform" = "true" ]; then
+        print_info "Cloning Sublime Platform repo..."
+        if ! git clone --depth=1 https://github.com/sublime-security/sublime-platform.git; then
+            print_error "Failed to clone Sublime Platform repo\n"
+            printf "Troubleshooting tips: https://docs.sublimesecurity.com/docs/quickstart-docker#troubleshooting\n\n"
+            printf "You may need to run the following command before retrying installation:\n\n"
+            printf "rm -rf ./sublime-platform\n"
+            exit 1
+        fi
+
+        cd sublime-platform || {
+            print_error "Failed to cd into sublime-platform"
+            exit 1
+        }
+    fi
+
+    if [ "$interactive" = "true" ] && [ ! -f "$CERTBOT_ENV_FILE" ]; then
+        printf "\nWould you like to setup SSL with LetsEncrypt? You must have a custom domain. (y/N): "
+        read -r enable_ssl </dev/tty
+    fi
+
+    if [ "$enable_ssl" = "y" ] || [ "$enable_ssl" = "Y" ] || [ "$enable_ssl" = "yes" ]; then
+        print_color "\nYou will need to perform some manual steps in order to enable SSL. Please follow the instructions" info
+        print_info "at https://docs.sublimesecurity.com/docs/quickstart-docker#ssl and re-run this script to finish setup.\n" info
+        exit 0
+    fi
```

Comment:
Should we have an else case here that tells the user that we're going to skip SSL setup? Might be redundant so genuinely up to you if you think it'll add value to the user.

Reply:
Added!